packages/corvid-e2e-tests/test/drivers/localEditorDriver.js

```diff
@@ -32,6 +32,7 @@ module.exports = page => {
     await page.waitForSelector(THEMED_TEXTS_SELECTOR);
     const textElements = await page.$(THEMED_TEXTS_SELECTOR);
     await textElements.click();
+    await page.waitFor(500);
```

Comment:
The termination of the window in tests was happening before the element was actually added.
TODO: find a better way to wait for the element to actually be added